can_tools/scrapers/nytimes/nyt_cases_deaths.py

```diff
@@ -120,6 +120,15 @@ class NYTimesCasesDeaths(FederalDashboard):
         "deaths": CUMULATIVE_DEATHS_PEOPLE,
     }
 
+    def __init__(self, execution_dt: pd.Timestamp = pd.Timestamp.utcnow()):
```

Comment:
add caching for the NYT scraper so that we can check the source every 30 minutes. This uses https://api.github.com/repos/nytimes/covid-19-data/commits as the URL due to [previous concerns](https://github.com/covid-projections/can-scrapers/pull/394#discussion_r782535124), which I think should get a new ETag whenever a new commit is made